sound/soc/sof/intel/hda-loader.c

```diff
@@ -58,7 +58,7 @@ hda_cl_prepare(struct device *dev, unsigned int format, unsigned int size,
 	struct hdac_stream *hstream;
 	int ret;
 
-	hext_stream = hda_dsp_stream_get(sdev, direction, 0);
+	hext_stream = hda_dsp_stream_get(sdev, direction, 0, true);
```

Comment:
I'm not a fan of these true/false parameters. Can we add a enum or define for this?

 ```
hext_stream = hda_dsp_stream_get(sdev, direction, 0, HDA_STREAM_USE_LINK_DMA);
```

is much more readable. I know we have many true/false interfaces in existing code, so if you @bardliao @ranj063 @ujfalusi are ok, I'm not going to block, but I do find these hard to read (in the calling code).